### ISSUE TITLE
[Feature]Maintainer of the site has access to delete the reported/flagged/spam blogs.

### ISSUE THREAD

**Issue by preranah7** [Anshul439/Blogverse#19]
Each site has their own maintainers.
For the authenticity of users who use the web application and the blogs which are getting provided to them for reading should be authentic.
The proposed solution is
The maintainers of the site can visit the site after a particular time span and can delete the blogs which are marked as spam or are flagged.
The user model will have a extra field named, role which will be the enum type containing fields as ['maintainer','user'].
The default value will be 'user' and the to make the value as 'maintainer', the maintainer of the site should be the first among all the users to signup when the site is deployed.
The second solution for making the the value as maintainer is directly accessing from the database and changing the value.
The user having the role as maintainer can be provided the access for delete blog, ensuring the users of the site gets authentic and real blogs.
Thankyou.

**Comment by Anshul439** [comment]
There are already 2 roles for user - user and admin. For now a user can be made admin directly from database. Also there is no such feature as spam blogs. So if you can add this feature first (labelling blog as spam by admin) then I can assign you this.

**Comment by preranah7** [comment]
Actually someone had already raised a issue regarding report blog feature, hence I didn't mention it.
But if you only want to label the blog as spam by admin itself and not other users then I can implement the solution.
Thankyou.

**Comment by Anshul439** [comment]
Makes sense. Will assign this to you after the spam feature is implemented by the other person.

**Comment by preranah7** [comment]
Sure.
